lib/resources/dstu2/allergy_intolerance.yaml

```diff
@@ -0,0 +1,211 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: AllergyIntolerance
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
+  required:
+  - update: 'Yes'
+  - create: <b>Invalid</b>
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "123412"}'
+  note:
+  - update: The id value must match the AllergyIntolerance/&lt;id> value.
+  - create: The id field <b>must not</b> be set when performing an update operation.
+
+- name: contained
+  required: 'No'
+  cardinality: 0..*
+  type: Resource
+  description: Contained, inline Resources.
+  example: |
+             {"contained": [
+                {"resourceType": "RelatedPerson",
+                 "id": "123",
+                 "relationship": {
+                    "coding": [
+                      {"system": "http://hl7.org/fhir/v3/RoleCode",
+                       "code": "SIGOTHR"}]}}]}
+  note: >
+          These should be either `Practitioner` or `RelatedPerson` resources that are referenced with the
+          [reporter] field.
+
+- name: onset
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Date(/time) when manifestations showed.
+  example: '{"onset: "2012-07-13T00:00:00Z"}'
+
+- name: recordedDate
+  required: 'Yes'
+  cardinality: 0..1
+  type: dateTime
+  description: When the sensitivity was recorded.
+  example: '{"recordedDate": "2015-10-14T13:13:20-06:00"}'
+
+- name: recorder
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Who recorded the sensitivity.
+  note: Recorder must be a `Practitioner` reference.
+  example: |
+             {"recorder": {
+                "reference": "Practitioner/21500971"}}
+
+- name: patient
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  description: Who the sensitivity is for.
+  example: |
+             {"patient": {
+                "reference": "Patient/5366327"}}
+
+- name: reporter
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference (Patient) | contained Reference (Practitioner | RelatedPerson)
+  description: Source of the information about the allergy.
+  example: |
+             Reporter as a Patient Reference
+               {"reporter": {
+                  "reference": "Patient/5366327"}}
+
+             Reporter as a contained reference
+               {"reporter": "#5366327}
```

Comment:
I bumped this up to 5366328 and changed the other contained id. It looks weird to have both a believably valid id and an obviously fake id in the same example.